README.md

```diff
@@ -74,11 +74,11 @@ It has the following properites, that are read only:
 
 **Source** represnts the sound playing in a 3d world, kind of like an abstract speaker.
 
-It has the following functions:
+Euphony has the following functions:
```

Comment:
I think this should be "Source has "... same for the rest of them.

Reply:
fixed